crates/frontend/src/components/context_card.rs

```diff
@@ -0,0 +1,80 @@
+use leptos::*;
+use serde_json::{Map, Value};
+
+use crate::{
+    components::{
+        condition_pills::{types::Condition, ConditionPills},
+        table::{types::Column, Table},
+    },
+    types::Context,
+};
+
+#[component]
+pub fn context_card(
```

Comment:
What does context_card look like? Do you have a screenshot?

Reply:
Its the same card which is there on the `Override` page.